frontend/pages/review.tsx

```diff
@@ -35,29 +37,88 @@ function ErrorWrapper({ message }: { message: string | undefined }) {
 }
 
 function ApprovalList({ category, filter }: { category: ApprovalCategory; filter: ApprovalFilterType }) {
+  const { approvals, isApprovalsLoading, isApprovalsError, mutateApprovals } = useListApprovals(category, filter)
+
+  const error = MultipleErrorWrapper(
+    `Unable to load approval page`,
+    {
+      isApprovalsError,
+    },
+    ErrorWrapper
+  )
+  if (error) return error
+
+  const getUploadCategory = (approvalCategory: ApprovalCategory) =>
+    approvalCategory === 'Upload' ? 'models' : 'deployments'
+
+  if (isApprovalsLoading || !approvals) {
+    return <Paper sx={{ mt: 2, mb: 2 }} />
+  }
+  return (
+    <Paper sx={{ mt: 2, mb: 2, pb: 2 }}>
+      <Typography sx={{ p: 3 }} variant='h4'>
+        {category === 'Upload' ? 'Models' : 'Deployments'}
+      </Typography>
+      {approvals.map((approval: Approval) => (
+        <ApprovalItem
+          approval={approval}
+          category={approval.approvalCategory}
+          version={approval.version as VersionDoc}
+          key={approval._id}
+          mutateApprovals={mutateApprovals}
+        />
+      ))}
+
+      {approvals.length === 0 && (
+        <EmptyBlob text={`All done! No ${getUploadCategory(category)} are waiting for approval.`} />
```

Comment:
Based on https://github.com/gchq/Bailo/pull/663/files#r1308684924 this should be updated. Also, to be super picky, saying "All done" makes the assumption that the user has reviewed something. They may have never reviewed something, but will still see this message. Due to this I've made a slight change in the wording:
```
<EmptyBlob text={`No ${uploadCategory}s awaiting approval.`} />
```